source/dub/dub.d

```diff
@@ -1884,6 +1893,7 @@ private struct UserConfiguration {
 
 	@Optional string[] registryUrls;
 	@Optional NativePath[] customCachePaths;
+	@Optional NativePath buildCacheDirectory;
```

Comment:
I'd like to re-think a bit the configuration, and have something a bit more segmented.
In this case, we could have:
```JSON
{
    "build": {
        "cache": "path/to/cache"
    }
}
```